fe1-web/src/features/witness/interface/Configuration.ts

```diff
@@ -21,27 +25,68 @@ export interface WitnessConfiguration {
    */
   getCurrentLao: () => WitnessFeature.Lao;
 
+  /**
+   * A hook returning the current lao id
+   * @returns The current lao id
+   */
+  useCurrentLaoId: () => Hash | undefined;
+
   /**
    * Returns whether the user is witness of the current lao
    * @returns Whether the user is witness of the current lao
    */
   isLaoWitness: () => boolean;
 
   /**
-   * Returns the currently active lao. Can only be used inside react components
-   * @returns The current lao
+   * Creates an action to add a notification to the redux store
+   * @returns A redux action that can be dispatched
+   */
+  addNotification: (notification: Omit<WitnessFeature.Notification, 'id'>) => AnyAction;
+
+  /**
+   * Creates an action that marks a message as read inside the redux store
+   * @returns A redux action that can be dispatched
    */
-  useCurrentLao: () => WitnessFeature.Lao;
+  markNotificationAsRead: (args: { laoId: string; notificationId: number }) => AnyAction;
+
+  /**
+   * Creates an action to discard a notification
+   * @returns A redux action that can be dispatched
+   */
+  discardNotifications: (args: { laoId: string; notificationIds: number[] }) => AnyAction;
 }
 
 /**
  * The type of the context that is provided to react witness components
  */
-export type WintessReactContext = Pick<WitnessConfiguration, 'useCurrentLao'>;
+export type WintessReactContext = Pick<
+  WitnessConfiguration,
+  | 'enabled'
```

Comment:
```suggestion
  'enabled'
```
Or does it work the way you wrote it?

Reply:
Okay, I find it weird but why not 😅